lib/led.js

```diff
@@ -23,27 +143,36 @@ function Led(opts) {
   var pinValue;
   var defaultLed;
   var state;
-  var isFirmata = true;
 
   if (!(this instanceof Led)) {
     return new Led(opts);
   }
 
+  var controller = null;
+  var err = null;
+
+  if (!opts || !opts.hasOwnProperty("controller")) {
+    controller = Controllers["Standard"];
+  }
+  else if (typeof opts.controller === "string") {
+    controller = Controllers[opts.controller];
+  }
+  else {
+    controller = opts.controller;
+  }
+
+  if (controller == null) {
+    controller = Controllers["Standard"];
+  }
+
+  Object.defineProperties(this, controller);
```

Comment:
See, I definitely can't speak to LED :-)

Would it be simpler have a default opts object and then decorate that with the user defined opts? That would simplify this and the defaultLed stuff.